Userland/Libraries/LibDebug/DebugSession.h

```diff
@@ -223,6 +238,45 @@ void DebugSession::run(DesiredInitialDebugeeState initial_debugee_state, Callbac
             state = State::FreeRun;
 
         auto regs = get_registers();
+
+        auto debug_status = peek_debug(6);
```

Comment:
Is there a constant we can use instead of `6`?

Reply:
I've moved them to constants in the ptrace header, that feels like the most applicable place to put them?